### PR TITLE
Use bunyan-syslog-udp instead of bunyan-syslog

### PR DESCRIPTION
**Pull request by d00rman**

### CODE REVIEW

**Comment by d00rman** [comment]
cc @wikimedia/services

File: lib/logger.js [hunk 1]
```diff
@@ -55,7 +55,6 @@ var streamConverter = {
             level: stream.level || conf.level,
             type: 'raw',
             stream: syslog_stream.createBunyanStream({
-                type: 'sys',
                 facility: syslog_stream.local0,
```
Comment (Pchelolo) [inline lib/logger.js]: We don't expose these as enum any more, so this should be `'local0'`

Comment (d00rman) [inline lib/logger.js]: Good catch!

**Comment by Pchelolo** [comment]
Important note: cluster+udp [is broken in node 0.12.7](https://github.com/nodejs/node-v0.x-archive/issues/9261), so syslog logging wouldn't work (and actually the whole app would crash). There's a workaround for that, but we could avoid introducing it, because 0.12.8 is coming pretty soon and we are not using 0.12 anywhere.

**Comment by Pchelolo** [comment]
LGTM

**Comment by d00rman** [comment]
Yeah, I think 0.12.7 being currently broken for this specific use-case is acceptable, especially since 0.12.8 will contain [the fix for it](https://github.com/nodejs/node-v0.x-archive/pull/8643).

**Pull request merged**